packages/headless/src/features/analytics/search-analytics-actions-loader.ts

```diff
@@ -402,13 +406,7 @@ export interface SearchAnalyticsActionCreators {
    *
    * @returns A dispatchable action.
    */
-  logSearchboxSubmit(): AsyncThunkAction<
-    {
-      analyticsType: AnalyticsType.Search;
-    },
-    void,
-    AsyncThunkAnalyticsOptions<StateNeededByAnalyticsProvider>
-  >;
+  logSearchboxSubmit(): temp__MakeAnalyticsActionReturnType<AnalyticsType.Search>;
```

Comment:
It could be handy to create a type e.g `SearchAnalyticsAction` or `SearchAnalyticsPayload` to avoid needing to import and specify the generic param.